recipes/tornado-sqlalchemy/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "tornado-sqlalchemy" %}
+{% set version = "0.5.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+#https://files.pythonhosted.org/packages/d9/4b/02fc7c444590a077aa4f0a5fbf8380be07d71a30fd435751fbfa4fa6a775/tornado_sqlalchemy-0.5.0.tar.gz
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/tornado_sqlalchemy-{{ version }}.tar.gz
+  sha256: bf00483143677e41dc7ed8c5131b4c8a290a005b7411c53a91700ddec658fb8d
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python >=3.5
+  run:
+    - python >=3.5
+    - sqlalchemy >=1.0
+    - tornado >=4.0
+
+test:
+  imports:
+    - tornado_sqlalchemy
+
+about:
+  home: https://github.com/siddhantgoel/tornado-sqlalchemy
+  license: MIT
+  license_family: MIT
+  license_file: 
+  summary: SQLAlchemy helpers for working in Tornado
+  doc_url: https://tornado-sqlalchemy.readthedocs.io/
+  dev_url: https://github.com/siddhantgoel/tornado-sqlalchemy
+
+extra:
+  recipe-maintainers:
+    - lincrosenbach
+    - kfasolin
+    - nicoddemus
```

Comment:
@kfasolin  and @nicoddemus need to confirm they are ok maintaining the recipe.

Reply:
Sure 👍